Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -mod=vendor -a
 
 #####################################################################################################
 # Build the operator image
-# note: update origin-cli-artifacts from `latest` to an appropriate OCP verison during release e.g `4.17`
+# note: update origin-cli-artifacts from `latest` to an appropriate OCP verison during release e.g `4.18`
```

Comment:
New release version is typically not available right away.  So it needs to remain latest